ref_test.go

```diff
@@ -70,3 +74,50 @@ func TestReference_EXTENSION_NO_EMPTY_LINE_BEFORE_BLOCK(t *testing.T) {
 	}
 	doTestsReference(t, files, NoEmptyLineBeforeBlock)
 }
+
+// benchResultAnchor is an anchor variable to store the result of a benchmarked
+// code so that compiler could never optimize away the call to runMarkdown()
+var benchResultAnchor string
+
+func BenchmarkReference(b *testing.B) {
+	params := TestParams{Options: Options{Extensions: NoExtensions}}
```

Comment:
How representative is this configuration of real world usage? Does using "common" flags have a significant effect on the performance?